packages/strapi-plugin-graphql/services/GraphQL.js

```diff
@@ -28,36 +28,57 @@ module.exports = {
       type === 'Int' ||
       type === 'Float' ||
       type === 'String' ||
-      type === 'Boolean'
+      type === 'Boolean' ||
+      type === 'DateTime'
```

Comment:
Don't forget the `scalar JSON`

Reply:
Can we group by a JSON field ?

Reply:
In fact, yes we can (but I'm not sure it was very usefull 😄)

Reply:
Emmm I need to try this